channels/us.m3u

```diff
@@ -3564,3 +3564,15 @@ http://wttv-lh.akamaihd.net:80/i/WTTVBreaking_1@333494/master.m3u8
 https://live.chdrstatic.com/cheddar/primary/4.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="Mohabat TV" tvg-country="" tvg-language="" tvg-logo="https://i.imgur.com/tucTC1O.jpg" group-title="",Mohabat TV
 http://media.mohabat.tv:1935/live_transcoder/ngrp:mohabat.stream_all/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="Radio Javan" tvg-country="" tvg-language="" tvg-logo="https://i.imgur.com/Ba2GkX3.jpg" group-title="",Radio Javan
+http://51.210.199.6/hls/stream.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Radio Javan
+https://stream.rjtv.tv/live/smil:rjtv.smil/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Radio Javan TV
+http://stream.rjtv.tv/live/smil:rjtv.smil/chunklist_b2892000.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Radio Javan TV
+https://stream.rjtv.tv/live/smil:rjtv.smil/playlist.m3u8
```

Comment:
Radio Javan is an Iranian channel: https://www.lyngsat.com/radiochannels/ir/Radio-Javan.html